client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -3,7 +3,6 @@
 import React, { Component } from 'react';
 import { compose } from 'redux';
 import { connect } from 'react-redux';
-import Griddle from 'griddle-react';
```

Comment:
Can we also remove Griddle from the `package.json` since we're not using it anymore?